io_scene_godot/converters/animation/serializer.py

```diff
@@ -493,7 +494,11 @@ def __init__(self, name, owner_anim_player):
         super().__init__('Animation', name)
         self['step'] = 0.1
         self['length'] = 0
-        self['loop'] = name.endswith("_loop")
+
+        suffix = (re.search("[-_]loop", name, re.IGNORECASE) or '')[0]
```

Comment:
You may want it to be `(re.search("[-_]loop", name, re.IGNORECASE) or [''])[0]`, otherwise it will have index error when there is no match.

Thanks for the fixing and good idea to throw the warning.

Reply:
Oops, good catch. Can't believe I didn't test that :man_facepalming: